### PR TITLE
Add 'router static vrf' resources

### PR DESCRIPTION
**Pull request by shebang42**
Add resources & data sources for:
- iosxr_router_static_vrf
- iosxr_router_static_vrf_ipv4_multicast
- iosxr_router_static_vrf_ipv4_unicast
- iosxr_router_static_vrf_ipv6_multicast
- iosxr_router_static_vrf_ipv6_unicast

Notes:
- Not sure if I interpreted `id: true` correctly.
- `make testacc` fails.  Also with the unmodified code base.  Same as:
  https://github.com/CiscoDevNet/terraform-provider-iosxr/pull/213#issuecomment-1829405429

### CODE REVIEW

**Review by danischm — CHANGES_REQUESTED** [review]

**Comment by danischm** [comment]
Thanks a lot for your contribution!

File: gen/definitions/router_static_vrf.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+name: Router Static VRF
```
[the remark this answers is not in the document]
Comment (shebang42) [reply gen/definitions/router_static_vrf.yaml]: You're correct.  I assumed I need to be explicit in the config.
Commit [9b1f1c1]

File: gen/definitions/router_static_vrf_address_family_ipv4_multicast.yaml [hunk 2]
```diff
@@ -0,0 +1,147 @@
+---
+name: Router Static VRF IPv4 Multicast
+path: Cisco-IOS-XR-um-router-static-cfg:/router/static/vrfs/vrf[vrf-name=%s]/address-family/ipv4/multicast/prefixes/prefix[prefix-address=%s][prefix-length=%d]
+doc_category: Routing
+attributes:
+  - yang_name: vrf-name
+    example: VRF2
+    id: true
```
[the remark this answers is not in the document]
Comment (shebang42) [reply gen/definitions/router_static_vrf_address_family_ipv4_multicast.yaml]: Commit [75be364]

**Comment by shebang42** [comment]
Note: In ~3 weeks, more PR's will follow for router_bgp.

**Review by danischm — CHANGES_REQUESTED** [review]
After removing the `router_static_vrf` definition, we need to remove the related examples and .go files in the internal/provider directory manually.

**Review by danischm — APPROVED** [review]

**Pull request merged**